### DISCUSSION TITLE
Chat input is hidden below the screen boundary on iPhone 11 Pro

### DISCUSSION THREAD

**Discussion by robacarp** [AllYourBot/hostedgpt#222] [General]
https://github.com/allyourbot/hostedgpt/assets/208647/d6774ca3-0335-4d65-808a-f6c16d7a1912

I cropped out most of my address bar, but left what I could without exposing my hosted app url.

**Comment by keithschacht** [comment]
@robacarp Ah, good catch. This is a legit issue. Converting this to an issue now. Thanks

**Comment by keithschacht** [comment]
Issue #223
